.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,160 @@
+name: Tests
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  php:
+    name: PHP Unit
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          cache: 'yarn'
+
+      - name: Setup WP-CLI
+        uses: godaddy-wordpress/setup-wp-cli@1
+
+      - name: Install dependencies and build
+        run: |
+          npm i -g yarn
+          composer install
+          yarn install --immutable
+          yarn build
+
+      - name: Setup wp-env
+        uses: godaddy-wordpress/setup-wp-env@main
+        with:
+          core: 'WordPress/WordPress#6.0'
+          phpVersion: '7.4'
+          plugins: '["https://downloads.wordpress.org/plugin/coblocks.2.24.4.zip","https://downloads.wordpress.org/plugin/woocommerce.6.9.4.zip"]'
```

Comment:
It's probably better to remove the version from the URLs, so that we always have the latest version, what do you think?

Reply:
If we want to ensure the latest stable is always used, WordPress.org has a download link for that:
https://downloads.wordpress.org/plugin/coblocks.latest-stable.zip
https://downloads.wordpress.org/plugin/woocommerce.latest-stable.zip